Makefile

```diff
@@ -24,6 +24,9 @@ KUBECONFIG ?= $(HOME)/.kube/config
 
 PROJECT_WORKING_DIR := /go/src/github.com/kiagnose/kubevirt-realtime-checkup
 
+E2E_TEST_TIMEOUT ?= 1h
+E2E_TEST_ARGS ?= $(strip -test.v -test.timeout=$(E2E_TEST_TIMEOUT) -ginkgo.v -ginkgo.timeout=$(E2E_TEST_TIMEOUT) $(E2E_TEST_EXTRA_ARGS))
```

Comment:
What is the meaning of the enclosing `$()` ?

Reply:
In a Makefile, the $( ... ) syntax is used to denote the use of a variable. It is a way to expand the value of a variable within a string or command.

Reply:
I don't understand why is it needed in that context.
What will be the difference if
```
E2E_TEST_ARGS ?= strip -test.v -test.timeout=$(E2E_TEST_TIMEOUT) -ginkgo.v -ginkgo.timeout=$(E2E_TEST_TIMEOUT) $(E2E_TEST_EXTRA_ARGS)
```
will be used?

Also, why is `?=` needed here?
AFAIU the semantics are that it could also be set from the outside.

Reply:
In the alternative you suggested without $(strip ...), if there are any leading or trailing spaces in the string, they would be included in the value of E2E_TEST_ARGS.
It's just a bit more cautious..

The `?=` in `E2E_TEST_ARGS ?=...` is there so that we could potentially override the E2E_TEST_ARGS from outside the Makefile. It allows for flexibility.

Reply:
Please see man 1 strip https://man7.org/linux/man-pages/man1/strip.1.html

Is the behavior you want is something like https://ioflood.com/blog/bash-trim-whitespace/#:~:text=Trimming%20whitespace%20in%20Bash%20is,all%20spaces%20in%20a%20string. ?

Reply:
no, that would remove all spaces. 
I checked and it is OK without the strip. 
If we see we need it we can put it back later.